### ISSUE TITLE
Dynmap not working on windows server 2008 standard since 0.30

### ISSUE THREAD

**Issue by dimitrifrom31** [webbukkit/dynmap#618]
Hi,

I have been using dynmap for months without issue on my windows 2008 x64 standard dedibox.
But the last working version is 0.29, after that it stopped working without any error in log, just not saying that it is binding as it should:
2012-02-20 12:19:38 [INFO] This server is running CraftBukkit version git-Bukkit-1.1-R4-b1938jnks (MC: 1.1) (Implementing API version 1.1-R4)
2012-02-20 12:19:38 [INFO] Preparing level "world"
2012-02-20 12:19:38 [INFO] Default game type: 0
2012-02-20 12:19:38 [INFO] Preparing start region for level 0 (Seed: 390474546406377638)
2012-02-20 12:19:39 [INFO] Preparing start region for level 1 (Seed: 4797978390463725661)
2012-02-20 12:19:39 [INFO] Preparing spawn area: 85%
2012-02-20 12:19:39 [INFO] [BukkitCompat] Loading BukkitCompat vr16B.
2012-02-20 12:19:39 [INFO] MCMACOMPAT r16B
2012-02-20 12:19:39 [INFO] McMyAdmin Compatibility Plugin version r16B started.
2012-02-20 12:19:39 [INFO] [dynmap] Loading dynmap v0.32-884.
2012-02-20 12:19:39 [INFO] [dynmap] Using Bukkit Permissions (superperms) for access control
2012-02-20 12:19:40 [INFO] [dynmap] Loaded 15 shaders.
2012-02-20 12:19:40 [INFO] [dynmap] Loaded 82 perspectives.
2012-02-20 12:19:40 [INFO] [dynmap] Loaded 12 lightings.
2012-02-20 12:19:40 [INFO] [dynmap] version 0.32-884 is enabled - core version 0.32-83
2012-02-20 12:19:40 [INFO] [dynmap] Loaded 3 maps of world 'world'.
2012-02-20 12:19:40 [INFO] [dynmap] Loaded 1 pending tile renders for world 'world
2012-02-20 12:19:40 [INFO] [dynmap] Loaded 2 maps of world 'world_the_end'.
2012-02-20 12:19:40 [INFO] [dynmap] Enabled
2012-02-20 12:19:40 [INFO] Server permissions file permissions.yml is empty, ignoring it
2012-02-20 12:19:40 [INFO] Done (2.609s)! For help, type "help" or "?"

I have tried for testing purposes to copy the server to my windows 2008 x64 web edition dedibox and there it works fine...

So basically it works on web but not on standard edition so I was wondering if there was any special requirement or anything that could explain this.

Any help is apreciated.
